src/Ryujinx.HLE/HOS/Applets/SoftwareKeyboard/InlineResponses.cs

```diff
@@ -32,7 +32,10 @@ private static uint WriteString(string text, BinaryWriter writer, uint maxSize,
             }
 
             writer.Write(bytes);
-            writer.Seek((int)maxSize - bytes.Length, SeekOrigin.Current);
+            if (bytes != null)
+            {
+                writer.Seek((int)maxSize - bytes.Length, SeekOrigin.Current);
+            }
```

Comment:
```suggestion

            if (bytes != null)
            {
                writer.Seek((int)maxSize - bytes.Length, SeekOrigin.Current);
            }

```

but this condition do any sense. There is `writer.Write(bytes);` just above, if `bytes` where `null` it will reach a NRE here, so the check will change anything.

Reply:
> but this condition do any sense. There is writer.Write(bytes); just above, if bytes where null it will reach a NRE here, so the check will change anything.

This is not resolved!